### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by skinkie**

### CODE REVIEW

**Comment by danielmewes** [comment]
Thanks for the PR @skinkie !

Have you signed out CLA yet? http://rethinkdb.com/community/cla/

**Comment by skinkie** [comment]
Yes I did.

File: 1-reql-language/geo-support/ruby.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ While there *is* a [circle] command, it approximates a circle by defining either
 
 # Geospatial indexes #
 
-To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/ruby/index_create/) command, setting the `geo` optional argument to `Prue`. In JavaScript, this would be:
+To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/ruby/index_create/) command, setting the `geo` optional argument to `True`. In Ruby, this would be:
```
Comment (danielmewes) [inline 1-reql-language/geo-support/ruby.md]: This should also probably be a lower-case `true`?
(We can also take care of that on our end after merge.)

File: 1-reql-language/geo-support/python.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ While there *is* a [circle] command, it approximates a circle by defining either
 
 # Geospatial indexes #
 
-To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/python/index_create/) command, setting the `geo` optional argument to `Prue`. In JavaScript, this would be:
+To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/python/index_create/) command, setting the `geo` optional argument to `True`. In Python, this would be:
 
 ```py
 r.table('sites').index_create('locations', {geo: true})
```
Comment (danielmewes) [inline 1-reql-language/geo-support/python.md]: On an unrelated note, It looks like the Python and Ruby examples are not properly translated (the opt arg style is wrong). Pinging @chipotle

**Comment by chipotle** [comment]
It looks like that example is definitely JS everywhere, although the other seem to be okay. I'll fix that as part of this.

**Pull request merged**